### PR TITLE
Update python-quiz.ts

### PR DESCRIPTION
**Pull request by shubhsinha**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by shubhsinha** [comment]
added a Python question

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1479,6 +1479,15 @@ const pythonQuiz = [
     Explanation: "Tkinter provides an object-oriented interface to the Tk GUI toolkit that is built-in to Python.",
     Link: "https://www.freecodecamp.org/news/learn-how-to-use-tkinter-to-create-guis-in-python/",
   },
+  {
+    Question: "What will be the output of command, round(0.6)-round(-0.6)?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1483]:
```suggestion
    Question: "In Python, what will be the output for the following code: print(round(0.6)-round(-0.6))?",
```

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1479,6 +1479,15 @@ const pythonQuiz = [
...
+    Answer: "2",
+    Distractor1: "0",
+    Distractor2: "1",
+    Distractor3: "-1",
+    Explanation: "round() function in Python, rounds off the number away from 0 to the nearest integer, round(0.6) = 1 and round(-0.6) = -1. So 1-(-1)=2.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1488]:
```suggestion
    Explanation: "In Python, round(0.6) will round up to 1 and round(-0.6) will round down to -1. The result for round(0.6)-round(-0.6) will be 2.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by shubhsinha** [comment]
updated as asked

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**